test/cli/help/validate-schema/expected-stdout

```diff
@@ -0,0 +1,13 @@
+Command "local validate schema"
+
+Validate content of the specified JSON file
+against the specified JSON schema file.
+
+The JSON schema should contain a schema for the "parameters" key,
+just like in a component definition.
+
+Usage:
+  %s local validate schema schema.json config.json [flags]
```

Comment:
A `validate schema` umoznuje specifikovat vlastnu JSON schemu.

**To moze pouzit component factory v CI, teda ze netestuje aktualnu schemu, ale novu schemu, ktoru chcu nastavit.**